pkg/components/components.go

```diff
@@ -146,6 +146,14 @@ func GetDeployment(version string, operatorVersion string, namespace string, rep
 										},
 									},
 								},
+								{
+									Name: "OPERAND_NAMESPACE",
```

Comment:
why we need this?

can't we just use the `OPERATOR_NAMESPACE`

Reply:
We could, but what if we would need to use two different namespaces then? If we keep it split like this, all our components will work with the assumption those namespaces are not always identical. Maybe I'm too paranoid though.